server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -42,4 +33,21 @@ describe('educationAndWorkPlanClient', () => {
       expect(nock.isDone()).toBe(true)
     })
   })
+
+  describe('getActionPlan', () => {
+    it('should get Action Plan', async () => {
+      // Given
+      const prisonNumber = 'A1234BC'
+      const systemToken = 'a-system-token'
+      const actionPlanResponse = aValidActionPlanResponseWithOneGoal()
+      educationAndWorkPlanApi.get(`/action-plans/${prisonNumber}`).reply(200, actionPlanResponse)
+
+      // When
+      const actualPlanDto = await educationAndWorkPlanClient.getActionPlan(prisonNumber, systemToken)
+
+      // Then
+      expect(nock.isDone()).toBe(true)
+      expect(actualPlanDto.prisonNumber).toEqual(prisonNumber)
```

Comment:
Any thoughts re: creating an expected action plan DTO, and asserting actualPlanDto equals the expected one? 🤔

Reply:
I considered that, but I didn't see the need to test the mapping here (above and beyond the test for the mapper itself).

Note that it would require creating a DTO with the exact same data that's in the `actionPlanResponse` (i.e. a different object), which seemed more trouble than it's worth.